### ISSUE TITLE
Issue with building Android Bundle

### ISSUE THREAD

**Issue by konradm** [joinflux/capacitor-segment#13]
Hey!
(Full disclosure: I'm a frontend designer with some frontend shaky coding knowledge 🙃)

When building, I got this error:
```
A problem occurred evaluating project ':joinflux-capacitor-segment'.
> Could not find method compile() for arguments [com.segment.analytics.android:analytics:4.+] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

I found [here](https://stackoverflow.com/questions/67366255/could-not-find-method-compile-for-arguments-com-sparkjavaspark-core2-9-3-o) that `compile` might be outdated. I've changed it to `implementation` and voilà, the build worked. Does this make sense at all? I barely know what I'm doing, ha. Appreciate your support!

https://github.com/joinflux/capacitor-segment/blob/c0454460737522cfd1103a23f13f7df42f4e13da/android/build.gradle#L54

**Comment by Nevvulo** [comment]
Hey @konradm, thanks for reaching out. I also had the same problem, we just merged #14 which changes the `compile` line to `api 'com.segment.analytics.android:analytics:4.+'`, used in the original fork and that seems to be working well for us. Can you try the latest version with Capacitor 4 and see if that helps? It'd also be awesome if we could get information around the Gradle/JDK version you're using, thanks!

**Comment by Nevvulo** [comment]
Going to close this as we haven't heard back in a bit, but please update us if there's still some problems and happy to re-open. As mentioned this issue should be fixed now 😄
